docs/docsite/rst/user_guide/playbooks_error_handling.rst

```diff
@@ -63,7 +63,9 @@ the handler from running, such as a host becoming unreachable.)
 Controlling What Defines Failure
 ````````````````````````````````
 
-Suppose the error code of a command is meaningless and to tell if there
+Ansible lets you define what "failure" means in each task. Ansible joins lists of multiple conditions for failure with an implicit ``and`` - if you want any one of multiple conditions to trigger a failure, you must define them in a string with an explicit ``or`` operator.
+
```

Comment:
i would separate the 'Ansible joins' and also prefix with 'As for all conditionals in Ansible' since this is a property of conditionals in general.

Reply:
Done. Thanks for the tip.